examples/adapters/modeling_tf_adapter_bert.py

```diff
@@ -0,0 +1,99 @@
+import tensorflow as tf
+import copy
+import numpy as np
+from transformers.modeling_tf_bert import gelu, gelu_new, swish, ACT2FN
+
+class AdapterModule(tf.keras.Model):
+  __instance_number = 0
+  __instance = None
+  @staticmethod 
```

Comment:
Why is there a need to keep the instance number here?

Reply:
According to Figure 2 in the paper, the Adapter module is added twice to each transformer layer before BERT's LayerNorms: one in the TFBertSelfOutput class and another in the TFBertOutput class. **Adapters in each layer are shared**. Therefore, I controlled sharing them via __instance_number. In this way, I only create a new Adapter module for those getInstance calls that come from TFBertSelfOutput classes and then return the last __instance rather than create a new Adapter for the second calls from TFBertOutput classes. ![alt text](https://i.ibb.co/VBnTMgT/figure2.png)